walkthroughs/week-2-server/examples/page-view-counter/src/main/java/com/google/sps/servlets/PageViewServlet.java

```diff
@@ -24,10 +24,14 @@
 @WebServlet("/page-views")
 public class PageViewServlet extends HttpServlet {
 
-  private int pageViews = 0;
+    /**
+     *
+     */
+    private static final long serialVersionUID = 1L;
```

Comment:
Do you need to serialize the class? Or you can just ignore the serialization warning on Eclipse. Java generates a serialization ID at compile time if the serialVersionUID is not specified.